config-windows.md

```diff
@@ -94,3 +94,42 @@ The following parameters can be specified:
         }
    }
 ```
+
+## <a name="configWindowsCredentialSpec" />Credential Spec
+
+You can configure a container's group Managed Service Accounts (gMSAs) via the OPTIONAL `credentialspec` field of the Windows configuration. For more information about gMSAs, see [Active Directory Service Accounts for Windows Containers][gMSAOverview]. For more information about tooling to generate a gMSA, see [Deployment Overview][gMSATooling]. The `credentialspec` MUST be a string containing an escaped JSON object.
```

Comment:
> I'm not sure what this should be in schema\config-windows.json though if not type string?

You can just declare it an object and leave it at that (backing RFC drafts [here][1] and [here][2]):

```
"credentialspec": {
    "id": "https://opencontainers.org/schema/bundle/windows/credentialspec",
    "type": "object"
}
```

> To be clear though - you are recommending to remove the example (even though technically accurate)?

Ideally we'd link to the backing Windows docs (which are necessary to satisfy “[implementation-defined][3]”.  Lacking that, I'm in favor of including as much information as we can as long as we're reasonably confident that it won't go stale during the useful lifetime of the next spec release.  If you expect the links and example you have now to be current for the lifetime of the 1.0 runtime spec, I'm fine with them.  If you expect they will go stale before runtime-spec 1.0 is archaic, I'd rather replace them with something generic enough to stay current.

[1]: https://tools.ietf.org/html/draft-wright-json-schema-01#section-4.2
[2]: https://tools.ietf.org/html/draft-wright-json-schema-validation-01#section-6.25
[3]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc5/spec.md#notational-conventions